new-pipeline/src/dist.js

```diff
@@ -9,9 +9,19 @@ var gPreviousFilterAllSelected = {};
 var gAxesList;
 var gAxesSelectors;
 var gNoneKey = "__none__";
+var changePending = false;
 
 indicate("Initializing Telemetry...");
-
+function stopProgress() {
```

Comment:
I'm not a big fan of using globals to reach from JS file to JS file. It's loose coupling that is very fragile.

I wonder, though, if there -is- any way to do this in one place. We may be restrained from that because of the way that `dashboards.js` "helpfully" initializes all the multiselects for us... and the way the multiselect library works.

If we can't do this cleanly, I'd prefer we not do it at all (as helpful as this change would be) :S

Maybe we'll have to rewrite some of the UI before this becomes easily possible. Can you think of a different way to do this that doesn't require skyhooking in some globals from dist.js back into dashboards.js? The dependency on those files generally points the other way (dist.js can depend on things present in dashboards.js, not the other way)

Reply:
> Can you think of a different way to do this that doesn't require skyhooking in some globals from dist.js back into dashboards.js? 

I'll try my best to think of something else.

Reply:
Thank you for your understanding. I'm sorry for being particular about this.

Reply:
No, problem. I can see why my changes are not optimal. The major trick of this change will be to trigger a function on closing the multiselect-menu. I think I can follow the code of the bootstrap-multiselect to see how this is triggered and that may be the key to my solution

Reply:
@chutten , Have a look now